subnet/local_manager.go

```diff
@@ -68,7 +68,14 @@ func (c watchCursor) String() string {
 }
 
 func NewLocalManager(config *EtcdConfig) (Manager, error) {
-	r, err := newEtcdSubnetRegistry(config, nil)
+	var err error
+	var r Registry
+
+	if config.UseV3API {
```

Comment:
I haven't done the part to choose between v2 and v3 yet, so this branch always uses v3